hypershift-operator/controllers/nodepool/nodepool_controller_test.go

```diff
@@ -526,11 +577,11 @@ status:
 						Namespace: namespace,
 					},
 					Data: map[string]string{
-						TokenSecretConfigKey: machineConfig2,
+						TokenSecretConfigKey: "---" + machineConfig2 + "\n---\n  \n  \n---\n" + machineConfig3,
```

Comment:
can we still have a single variable defining both configs so it's immediately visible what a consumer would need to setup?